engine/script/src/script_extensions.cpp

```diff
@@ -29,7 +29,7 @@ namespace dmScript
     #define BIT_INDEX(b) ((b) / sizeof(uint32_t))
     #define BIT_OFFSET(b) ((b) % sizeof(uint32_t))
 
-    const char* EXTENSIONS_CONTEXT_NAME = "__extensions_context__";
+    const char EXTENSIONS_CONTEXT_NAME[] = "__extensions_context__";
```

Comment:
With this change, `sizeof` will return the size of the array rather than the size of the pointer. This is needed for using `sizeof` together with `lua_pushlstring` below.